addons/project/static/src/scss/project_form.scss

```diff
@@ -30,6 +30,15 @@
     }
 }
 
+.o_field_widget {
+    &.o_field_subtasks_one2many {
+        width: 100%;
+        > div {
+            width: 100%;
+        }
```

Comment:
For the rule above, I'm OK.
But why the direct descendant `<div>` ? :thinking:

Reply:
This is done for the original O2M/M2M fields styling inside forms, i kept it there, guess there are some condition where we need this to insure child div will expand into the parent, even tho after short testing the button still expands without the div selector even when resizing, do you think i should remove it?

https://github.com/odoo/odoo/blob/89e05fb8e96e1165241b91b825dd64a423d0dc16/addons/web/static/src/views/form/form_controller.scss#L704-L711

Thanks again

Reply:
If you don't need it... don't add it :wink: 

Actually, I'm not even sure that theO2M/M2M fields still need this rule...